pages/common/mogrify.md

```diff
@@ -0,0 +1,23 @@
+# Mogrify
```

Comment:
Hmm, I changed it to capital after bot complained, I must have misread the error message.
If we always start with lowercase, maybe add it also to bot's check?